RELEASEME.md

```diff
@@ -0,0 +1,99 @@
+ * to learn how to use sonatype with your credentials, check [this](http://www.scala-sbt.org/0.13/docs/Using-Sonatype.html#Fourth+-+Adding+credentials)
+
+### SBT-PGP Plugin
+
+ * add `addSbtPlugin("com.typesafe.sbt" % "sbt-pgp" % "0.8")`to your project's build.sbt, this allows you to sign and publish your artefacts by running publishSigned in sbt
+ 
+
+### Sonatype Credentials File
+
+ * if not existing, create ~/.sbt/0.13/sonatype.sbt
+ * edit above file and add
+  ```credentials += Credentials("Sonatype Nexus Repository Manager",
+                                "oss.sonatype.org",
+                                "<your username>",
+                                "<your password>")
+  ```
+
+### GPG SBT-File
+
+ * if not existing, create ~/.sbt/0.13/plugins/gpg.sbt
+ * edit above file and add sbt-pgp plugin with `addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")` (for sbt version 0.13.5 and above)
+   or with `addSbtPlugin("com.typesafe.sbt" % "sbt-pgp" % "0.8.3")` (for older sbt versions)
+
+## Preperations
+
+ * Check [GitHub](https://github.com/Galeria-Kaufhof/ha-jobs) for eventually open pull requests which should be mentioned in the release
+ * If there are open pull requests, check out and test them first, when they are OK you may merge them into the master
+ * Checkout/Clone the data status you want to release to your local repository
+ * if you wanna change some code before the release, do so and use `git push` to add them to github before you start with an release
+ * btw. builds from this project are made with travis, you can see an overview [here](https://travis-ci.org/Galeria-Kaufhof/ha-jobs) 
+ 
+
+## Snapshot Release
+
+ * Instead of the final release, a snapshot release can be changed and released several times
+ * a SNAPSHOT will NOT be released at maven central but at sonatype, where it can be used from
+ * To build a SNAPSHOT go to your project build.sbt and change the projectVersion by increment the version number and 
+   add -SNAPSHOT behind (e.g. `projectVersion = "1.2.4-SNAPSHOT"`)
+ * there are also version numbers mentioned in the README.md, change them also to the version number you used in the build.sbt
+ * start sbt on your project root
+ * publish the changes with `very publishSigned`
+ * confirm the publish with the your gpg password when sbt asks you
+ 
+
+## Final release
+
+ * the final public release will go out to maven central and, therefore, cannot be released twice with the same version number
+ * to build a regular release just increment the version number in your project build.sbt AND the README.md without adding
+   -SNAPSHOT at the end. If you released a SNAPSHOT with this code stand before do not incremnt the verison number, just remove the
+   -SNAPSHOT ending
+ * start sbt on your project root
+ * publish the changes with `very publishSigned`
+ * confirm the publish with the your gpg password when sbt asks you
+ 
+
+## Sonatype
```

Comment:
The following are steps of the final release, so this heading can be removed or should be h3 at least.